kbhff/api/email.py

```diff
@@ -1,6 +1,36 @@
 import easyimap, email
 from kbhff.api.exceptions import *
 import time
+import imaplib
+import sys
+
+class GmailConnection:
+    def __init__(self):
+        self.connection = None
+        self.mail_credentials = get_mail_credentials()
+        self.connect()
```

Comment:
It's often considered bad practice to do non-trivial things in constructors; so it might be better to expect the user to run `connect()` on a constructed Gmail instance when they want it to actually connect.